test/models/v3/getPenaltyDetails/FailureResponseSpec.scala

```diff
@@ -28,7 +30,7 @@ class FailureResponseSpec extends SpecBase {
       |}
       |""".stripMargin)
 
-  val model: FailureResponse = FailureResponse(
+  val model: FailureResponse = v3.FailureResponse(
```

Comment:
This file needs to be moved to the v3 package

Reply:
FailureResponse is already in the v3 package?